brian2tools/mdexport/mdexporter.py

```diff
@@ -102,19 +106,26 @@ def build(self, direct_call=True, debug=False, expander=None,
             
             # Check if Pandoc is installed
             try:
-                subprocess.check_call(["pandoc"])
+                subprocess.check_call(["pandoc", "--version"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
                 formats_extensions = {'latex':'.tex', 'html':'.html', 'pdf':'.pdf'}
-                if format == "all":
-                    formats = ['latex', 'html', 'pdf']
+                if type(Additionalformat) == str:
+                    if Additionalformat == "all":
+                        formats = ['latex', 'html', 'pdf']
+                    else:
+                        formats = [Additionalformat]
                 else:
-                    formats = format
+                    formats = Additionalformat if Additionalformat is not None else []
                 for format_name in formats:
                     filename = self.filename + formats_extensions[format_name]
                     subprocess.run(["pandoc", "--from", "markdown", "--to", format_name, "-o", filename, source_file])
-                    print("Conversion complete! Files generated:", filename)   
+                    print("Conversion complete! Files generated:", filename) 
+                   
             except subprocess.CalledProcessError:
                 raise Exception("Pandoc is not installed. Please install Pandoc and try again.")
```

Comment:
If `pandoc` is not installed, the error below will be triggered instead of this line. This exception here would currently only be raised if `pandoc` is installed, but `pandoc --version` fails nevertheless (not quite sure how this could happen).